gh-actions/go/gotestfmt/action.yaml

```diff
@@ -0,0 +1,35 @@
+name: Install gotestfmt and our wrapper script
+description: Install gotestfmt and our wrapper script which improves the output and copies it to log files
+
+inputs:
+  tools-directory:
+    description: Directory pointing to go.mod file for checking tool versioning. If none is provided, the latest version will be downloaded.
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install gotestfmt
+      working-directory: ${{ inputs.tools-directory }}
+      run: |
+        set -eu
+        TOOL=github.com/gotesttools/gotestfmt/v2/cmd/gotestfmt
+        VERSION=$(go mod edit --json | jq -r ".Require[] | select(.Path==\"${TOOL}\") | .Version" || true)
+
+        if [ -z "${VERSION}" ]; then
+          VERSION="latest"
```

Comment:
We can still rely on an input for that?

So we've both ways

Reply:
Can we add that when we actually want to use it? We now have one way to select the gotestfmt version which I'm not even sure we want to use, I would prefer to keep it at that until we actually have the need to support the input as well.

Reply:
Fully agreed with @adombeck. I’m not in favour for an option unless this is really needed and there is no other way. The matrix of complexity will ever increase and we will not be sure of the desired behaviour. Also, it means that many workflows will end up using different strategies, which will make maintainance harder in the longer term.